src/views/product-downloads-view/helpers.ts

```diff
@@ -75,6 +74,51 @@ export const generateDefaultPackageManagers = (
 	]
 }
 
+export function generateEnterprisePackageManagers(
+	product: Pick<ProductData, 'slug'>
+): PackageManager[] {
+	const productSlug = product.slug
+
+	return [
+		{
```

Comment:
💭 I wonder if it would make sense to lift this to a data file. Unless we're confident this package mgr data will continue to be consistent between products. Probably something we could just do later if needed.

Reply:
These are the values we _don't_ want to be overwritten/authorable because they're "global" values that affect all products. These can be overwritten though through the data files!

Reply:
Got it, I wasn't aware these were global values. so this makes sense 👌